providers/hubspot/read.go

```diff
@@ -60,8 +59,14 @@ func (c *Connector) Read(ctx context.Context, config common.ReadParams) (*common
 	} else {
 		// If NextPage is not set, then we're reading the first page of results.
 		// We need to construct the query and then make the request.
-		relativeURL := strings.Join([]string{"objects", config.ObjectName, "?" + makeQueryValues(config)}, "/")
-		rsp, err = c.Client.Get(ctx, c.getURL(relativeURL))
+		relativeURL := strings.Join([]string{"objects", config.ObjectName + "/"}, "/")
```

Comment:
This line can now be simplified to the following, which is more readable:
```
relativeURL := "objects/" + config.ObjectName
```

(I omitted the last `/` since I'm assuming c.getURL handles that. )

Reply:
I kept that there because it's emulating what the old code did. I tried to keep it exact.

Reply:
but I did simplify it, per your suggestion